vendor/init.go

```diff
@@ -113,13 +113,9 @@ func main() {
 	ctx := context.Background()
 	wg := sync.WaitGroup{}
```

Comment:
nit: Since it is not used until later, move the definition of wg down below the next condition, before the loop.

Reply:
All set.